operator/src/main/java/oracle/kubernetes/operator/helpers/PodDefaults.java

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at
+// http://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator.helpers;
+
+import static oracle.kubernetes.operator.KubernetesConstants.*;
```

Comment:
do not use wildcard imports

Reply:
Updated. What is the reason for that rule?